src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -172,6 +196,37 @@ public FormValidation doCheckRegion(@QueryParameter String region){
         }
         return ret;
     }
-    
-    
+
+    @RequirePOST
+    public FormValidation doValidateS3BucketConfig(@QueryParameter String container, @QueryParameter String prefix,
+                                                   @QueryParameter String region){
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        FormValidation ret = FormValidation.ok("success");
+        try {
+            S3BlobStore provider = new S3BlobStoreTester(container, prefix, region);
+            JCloudsVirtualFile jc = new JCloudsVirtualFile(provider, container, "");
+            jc.list();
+        } catch (Throwable t){
+            String msg = processExceptionMessage(t);
+            ret = FormValidation.error(StringUtils.abbreviate(msg, 200));
+            LOGGER.severe(t.getMessage());
```

Comment:
Get rid of that. We do not want system log messages appearing if the admin has not actually _saved_ a bad config!

Reply:
we can move it to FINEST for debug purposes

Reply:
Sure.